cmd/crio/main.go

```diff
@@ -131,9 +131,6 @@ func mergeConfig(config *server.Config, ctx *cli.Context) error {
 	if ctx.GlobalIsSet("pids-limit") {
 		config.PidsLimit = ctx.GlobalInt64("pids-limit")
 	}
-	if ctx.GlobalIsSet("enable-shared-pid-namespace") {
```

Comment:
The man pages need updating too (e.g. [here][1]).

[1]: https://github.com/kubernetes-incubator/cri-o/blame/0b736bb43fb9f6ffa29b477be9e292bec3686ef4/docs/crio.8.md#L97

Reply:
Updated the man page. Thanks for catching that!

Reply:
[Other man page][1] too ;).

[1]: https://github.com/kubernetes-incubator/cri-o/blame/0b736bb43fb9f6ffa29b477be9e292bec3686ef4/docs/crio.conf.5.md#L90

Reply:
Fixed. Removed it from helpers.bash as well. I think I got all of them now :)